dttools/src/jx_parse.c

```diff
@@ -270,6 +270,10 @@ static jx_token_t jx_scan( struct jx_parser *s )
 	} else if(c=='!') {
 		char d = jx_getchar(s);
 		if(d=='=') return JX_TOKEN_NE;
```

Comment:
Let's just make this a simple `else` and avoid the problem of understanding all the valid characters that may follow..
  (For example, an open paren could come after `!`)